site/content/en/docs/tasks/troubleshooting/troubleshooting_jobs.md

```diff
@@ -60,16 +62,29 @@ Job is called `my-job` in the `my-namespace` namespace.
    job-my-job-19797   user-queue    cluster-queue True       9m45s
    ```
 
-## Is my Job running?
+Once you have identified the name of the Workload, you can obtain all details by
+running the following comand:
+
+```bash
+kubectl describe workload -n my-namespace job-my-job-19797
+```
 
-To know whether your Job is running, look for the value of the `.spec.suspend` field, by
+## Is my Job suspended?
+
+To know whether your Job is suspended, look for the value of the `.spec.suspend` field, by
```

Comment:
Maybe also include info how to tell that the job is suspended by looking at the corresponding workload object.

Reply:
That's the next question: Is my job admitted?